statsmodels/tools/grouputils.py

```diff
@@ -403,6 +403,7 @@ def get_slices(self, level=0):
         """
         # TODO: refactor this
         groups = self.index.get_level_values(level).unique()
+        groups = np.array(groups)
```

Comment:
Sorry for missing this! Just realized this has been sitting here. Our of curiosity, why is this better? Would it maintain some of the pandas functionality? 

Happy either way.